db/migrate/20161004074754_add_notifivations_settings_to_user.rb

```diff
@@ -0,0 +1,13 @@
+class AddNotifivationsSettingsToUser < ActiveRecord::Migration
```

Comment:
Grammar Nazi!!!oneone
`AddNotificationsSettingsToUser`, NOT `AddNotifivationsSettingsToUser`